opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "ocamlbuild"
+maintainer: "Gabriel Scherer <gabriel.scherer@gmail.com>"
+
+authors: [
+  "Nicolas Pouillard"
+  "Berke Durak"
+]
+
+license: "LGPL-2 with OCaml linking exception"
+dev-repo: "https://github.com/ocaml/ocamlbuild.git"
+homepage: "https://github.com/ocaml/ocamlbuild/"
+bug-reports: "https://github.com/ocaml/ocamlbuild/issues"
+
+build: [
+  [make]
+  [make "ocamlbuild.native"]
+]
+
+# TODO
+# install: [ ]
+
+doc: [
+  "http://caml.inria.fr/pub/docs/manual-ocaml/ocamlbuild.html"
+  "https://github.com/gasche/manual-ocamlbuild/blob/master/manual.md"
+]
+
+available: [ocaml-version >= "3.10"]
```

Comment:
OK that makes sense. It's a bit unsatisfactory since I'm sure ocamlbuild can be compiled with ocaml older than 4.02.3 but installing it might interfere with what's already provided.

There's also the option of specifying a conflict with OCaml `< 4.03` but I don't know how that's any better.